.mailmap

```diff
@@ -1,114 +1,53 @@
-Valentina Staneva <val_is_everywhere@yahoo.com> <vms16@uw.edu>
-Jim Stapleton <jmstpltn@gmail.com>
-Brian Stucky <stuckyb@flmnh.ufl.edu>
-Cody Taylor <codyalantaylor@gmail.com>
 Bartosz Telenczuk <bartosz@telenczuk.pl>
-Ian Thomas <ianedwardthomas@gmail.com>
-Brian Thorne <hardbyte@gmail.com>
+Brian Stucky <stuckyb@flmnh.ufl.edu>
+Damien Irving <damien_irving@yahoo.com.au> <irving.damien@gmail.com>
+Derek Howard <derekhoward@gmail.com>
+Edwin van der Helm <vdhelm@strw.leidenuniv.nl> <vdhelm@users.noreply.github.com>
+Evan P. Williamson <evanpeterw@gmail.com>
+Ewan Barr <ewan.d.barr@googlemail.com>
+François Michonneau <francois.michonneau@gmail.com>
+François Michonneau <francois.michonneau@gmail.com> <francois.michonneau@gmail.Com>
 Giovanni Torres <giovanni.torres@gmail.com>
-Tim Tröndle <tim.troendle@younicos.com>
-Olav Vahtras <vahtras@kth.se>
-Edwin van der Helm <vdhelm@strw.leidenuniv.nl>
-Alistair Walsh <alistair.walsh@me.com>
-Derek Weitzel <djw8605@gmail.com>
-Daniel Wheeler <daniel.wheeler2@gmail.com>
-Ethan White <ethan@weecology.org>
-Frank Willmore <frankwillmore@gmail.com>
 Greg Wilson <gvwilson@software-carpentry.org> <gvwilson@third-bit.com>
-Andrea Zonca <code@andreazonca.com>
+Ian Carroll <icarroll@sesync.org> <itcarroll@users.noreply.github.com>
+James Adams <jladams@ou.edu>
+James Allen <james@sharelatex.com> <jamesallen0108@gmail.com>
+Jean-Christophe Leyder <jc.leyder@sciops.esa.int> <leyder@users.noreply.github.com>
+Jessica Upani <nabbygirl@gmail.com>
+Jim Stapleton <jmstpltn@gmail.com>
+Jin Choi <jinchoi721@gmail.com>
+John Chase <chasejohnh@gmail.com>
+Kate Hertweck <k8hertweck@gmail.com>
+Leigh Sheneman <leighs@msu.edu>
+Maxime Boissonneault <maxime.boissonneault@calculquebec.ca>
+Mike Jackson <m.jackson@software.ac.uk> <michaelj@epcc.ed.ac.uk>
+Narayanan Raghupathy <narayanan.r@gmail.com>
+Neal Davis <davis68@illinois.edu> <davisneale@gmail.com>
+Nicola Soranzo <nicola.soranzo@tgac.ac.uk> <nsoranzo@tiscali.it>
+Pauline Barmby <pbarmby@uwo.ca>
+Pawel Pomorski <ppomorsk@users.noreply.github.com>
+Peter Cock <p.j.a.cock@googlemail.com>
+Raffaella Demichelis <raffaella.demichelis@curtin.edu.au>
+Raniere Silva <raniere@rgaiacs.com> <ra092767@ime.unicamp.br>
+Raniere Silva <raniere@rgaiacs.com> <raniere@ime.unicamp.br>
+Rémi Emonet <remi@heeere.com> <remi.emonet@reverse--com.heeere>
+Rémi Emonet <remi@heeere.com> <twitwi@users.noreply.github.com>
+Richard Clare <richard.clare@canterbury.ac.nz>
+Ryan Dale <dalerr@niddk.nih.gov>
+Ryan Neufeld <raeneufe@uwaterloo.ca> <metavee@users.noreply.github.com>
+Sarah Clayton <sarah.c.clayton@gmail.com>
+Sigrid Klerke <SigridK@users.noreply.github.com>
+Timothée Poisot <t.poisot@gmail.com> <tim@poisotlab.io>
+Toby Hodges <tbyhdgs@gmail.com>
+Trevor Bekolay <tbekolay@gmail.com>
+Valentina Staneva <valentina-s@users.noreply.github.com>
+Valentina Staneva <val_is_everywhere@yahoo.com>
+Willem Ligtenberg <willem@wligtenberg.nl>
+YourName <you@example.com>
```

Comment:
There is actually a commit from "Your Name <you@example.com>".
I added this rewrite so that it can be detected later in the release process ("YourName" is single-word so it won't pass the "firstName lastName" check and will ring a bell in the release process).